vm/onboarding.go

```diff
@@ -43,17 +54,30 @@ func postHTTPRequest(eventData []byte, vmAction string, address string) (string,
 
 // List - Lists all configured VMs
 func List(address string) error {
+	var endpoints []KVMSEndpoint
 
 	vmlist, err := postHTTPRequest(nil, "vmlist", address)
 	if err != nil {
 		fmt.Println("Failed to get vm list")
 		return err
 	}
 
-	if vmlist == "" {
+	err = json.Unmarshal([]byte(vmlist), &endpoints)
```

Comment:
@wazir-ahmed Instead of using encoding/json import, kindly use clarketm/json formatting. 
This import ignores all empty elements in DB. 

You can refer to https://github.com/kubearmor/kubearmor-client/blob/c9431d1b17247692627f6ed8b3e8af6a1b2fc2ce/discover/discover.go#L12

Reply:
I'm only doing `json --> struct` conversion here. So the standard `json` library works fine in this case.